TEST_SCRIPTS_README.md

```
@@ -0,0 +1,70 @@
+# Test Scripts for Mad Spark Alt
+
+This directory contains test scripts demonstrating the Mad Spark Alt QADI system functionality.
+
+## Prerequisites
+
+Make sure Mad Spark Alt is properly installed:
+```bash
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)